content/api/commands/getallcookies.md

```diff
@@ -0,0 +1,105 @@
+---
+title: getAllCookies
+---
+
+Get all browser cookies.
+
+## Syntax
+
+```javascript
+cy.getAllCookies()
+cy.getAllCookies(options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+cy.getAllCookies() // Get all cookies
+```
+
+### Arguments
+
+**<Icon name="angle-right"></Icon> options** **_(Object)_**
+
+Pass in an options object to change the default behavior of
+`cy.getAllCookies()`.
+
+| Option    | Default                                                        | Description                                                                              |
+| --------- | -------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `log`     | `true`                                                         | Displays the command in the [Command log](/guides/core-concepts/cypress-app#Command-Log) |
+| `timeout` | [`responseTimeout`](/guides/references/configuration#Timeouts) | Time to wait for `cy.getAllCookies()` to resolve before [timing out](#Timeouts)          |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+`cy.getAllCookies()` yields an array of cookie objects. Each cookie object has
+the following properties:
+
```

Comment:
maybe its common knowledge, but should we specify which are strings vs boolean vals?

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4918/commits/be1297a20cc599c86678af0a7e06ad5429d7cf0b for this and `getCookies`